From ca6798717a49fa865770f6033dd427c51781339c Mon Sep 17 00:00:00 2001
From: Haoran Wang <b50027@freescale.com>
Date: Wed, 2 Dec 2015 10:55:11 +0800
Subject: [PATCH 572/602] MA-7242 Change the bt device group to net_bt_stack

Followed AOSP changes for the bt device group changing.
This change will fix the BT device multi-user share
permission issue.

Signed-off-by: Haoran Wang <b50027@freescale.com>
---
 imx6/etc/ueventd.freescale.rc | 6 +++---
 imx7/etc/ueventd.freescale.rc | 4 ++--
 sabresd_6dq/init.rc           | 2 +-
 sabresd_6sx/init.rc           | 2 +-
 sabresd_7d/init.rc            | 2 +-
 5 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/imx6/etc/ueventd.freescale.rc b/imx6/etc/ueventd.freescale.rc
index db57bd9..2f8226e 100644
--- a/imx6/etc/ueventd.freescale.rc
+++ b/imx6/etc/ueventd.freescale.rc
@@ -1,6 +1,6 @@
 /dev/pmem_gpu             0660   system     graphics
-/dev/ttymxc2              0660   bluetooth  bluetooth
-/dev/ttymxc4              0660   bluetooth  bluetooth
+/dev/ttymxc2              0660   bluetooth  net_bt_stack
+/dev/ttymxc4              0660   bluetooth  net_bt_stack
 /dev/snd/*                0660   system     audio
 /dev/ttyUSB*              0640   radio      radio
 /dev/ttyACM*              0640   radio      radio
@@ -11,7 +11,7 @@
 /dev/video16              0660   system     graphics
 /dev/mxc_ipu              0660   media      system
 /dev/mxc_vpu              0660   media      system
-/dev/uinput               0660   system     bluetooth
+/dev/uinput               0660   system     net_bt_stack
 /dev/ion                  0666   media      system
 /dev/galcore	          0666 	 system     graphics
 /dev/graphics/fb0         0660   system     graphics
diff --git a/imx7/etc/ueventd.freescale.rc b/imx7/etc/ueventd.freescale.rc
index 5007404..4a828d2 100644
--- a/imx7/etc/ueventd.freescale.rc
+++ b/imx7/etc/ueventd.freescale.rc
@@ -1,5 +1,5 @@
 /dev/pmem_gpu             0660   system     graphics
-/dev/ttymxc5              0660   bluetooth  bluetooth
+/dev/ttymxc5              0660   bluetooth  net_bt_stack
 /dev/snd/*                0660   system     audio
 /dev/ttyUSB*              0640   radio      radio
 /dev/ttyACM*              0640   radio      radio
@@ -10,7 +10,7 @@
 /dev/video16              0660   system     graphics
 /dev/mxc_ipu              0660   media      system
 /dev/mxc_vpu              0660   media      system
-/dev/uinput               0660   system     bluetooth
+/dev/uinput               0660   system     net_bt_stack
 /dev/ttymxc2              0660   system     gps
 /dev/ttya0                0660   system     gps
 /dev/ptya0                0660   system     gps
diff --git a/sabresd_6dq/init.rc b/sabresd_6dq/init.rc
index 3b66327..6fdd637 100755
--- a/sabresd_6dq/init.rc
+++ b/sabresd_6dq/init.rc
@@ -74,7 +74,7 @@ on boot
 
     # Prepare for bluetooth
     chmod 665 /sys/class/rfkill/rfkill0/state
-    chown bluetooth bluetooth /sys/class/rfkill/rfkill0/state
+    chown bluetooth net_bt_stack /sys/class/rfkill/rfkill0/state
     write /sys/class/rfkill/rfkill0/state 0
 
     # Prepare for RIL
diff --git a/sabresd_6sx/init.rc b/sabresd_6sx/init.rc
index fb7d68b..b6e0309 100755
--- a/sabresd_6sx/init.rc
+++ b/sabresd_6sx/init.rc
@@ -75,7 +75,7 @@ on boot
 
     # Prepare for bluetooth
     chmod 665 /sys/class/rfkill/rfkill0/state
-    chown bluetooth bluetooth /sys/class/rfkill/rfkill0/state
+    chown bluetooth net_bt_stack /sys/class/rfkill/rfkill0/state
     write /sys/class/rfkill/rfkill0/state 0
 
     # Prepare for RIL
diff --git a/sabresd_7d/init.rc b/sabresd_7d/init.rc
index 28a02c0..34c4a1f 100644
--- a/sabresd_7d/init.rc
+++ b/sabresd_7d/init.rc
@@ -77,7 +77,7 @@ on boot
 
     # Prepare for bluetooth
     chmod 665 /sys/class/rfkill/rfkill0/state
-    chown bluetooth bluetooth /sys/class/rfkill/rfkill0/state
+    chown bluetooth net_bt_stack /sys/class/rfkill/rfkill0/state
     write /sys/class/rfkill/rfkill0/state 0
 
     # Prepare for RIL
-- 
1.8.0

